docs/settings/monitoring-settings.asciidoc

```diff
@@ -62,14 +62,17 @@ If not set, {kib} uses the value of the <<elasticsearch-user-passwd, `elasticsea
 `monitoring.ui.elasticsearch.password`::
 Specifies the password used by {kib} monitoring to establish a persistent connection
 in {kib}  to the {es} monitoring cluster and to verify licensing status on the {es}
-monitoring cluster.
+monitoring cluster when using `monitoring.ui.elasticsearch.hosts`.
 +
-Every other request performed by *{stack-monitor-app}* to the monitoring {es}
+All other requests performed by *{stack-monitor-app}* to the monitoring {es}
 cluster uses the authenticated user's credentials, which must be the same on
 both the {es} monitoring cluster and the {es} production cluster.
 +
 If not set, {kib} uses the value of the <<elasticsearch-user-passwd, `elasticsearch.password`>> setting.
 
+`monitoring.ui.elasticsearch.serviceAccountToken`::
+Specifies a {ref}/security-api-create-service-token.html[service account token] for the {es} cluster where your monitoring data is stored when using `monitoring.ui.elasticsearch.hosts`.  This setting is an alternative to using `monitoring.ui.elasticsearch.username` and `monitoring.ui.elasticsearch.password` and cannot be used simultaneously. 
```

Comment:
The docs for `elasticsearch.serviceAccountToken` state this:

> If your Elasticsearch is protected with basic authentication, this token provides the credentials that the Kibana server uses to perform maintenance on the Kibana index at startup. This setting is an alternative to `elasticsearch.username` and `elasticsearch.password`.

I think the "and cannot be used simultaneously" added to the end of this sentence is a little confusing. I'd suggest either removing it (my preference), or splitting that into a separate sentence noting that this cannot be used while username/password are set.

```suggestion
Specifies a {ref}/security-api-create-service-token.html[service account token] for the {es} cluster where your monitoring data is stored when using `monitoring.ui.elasticsearch.hosts`.  This setting is an alternative to using `monitoring.ui.elasticsearch.username` and `monitoring.ui.elasticsearch.password`.
```